### PR TITLE
PR: 6635, 6881, 6888, 6900, 6955

### PR DESCRIPTION
**Pull request by peiying2**
*PR 6881: new page mozorg/about-2019.lang (can leverage some strings here and there)
*PR 6888: update firefox/new/quantum.lang (can leverage from firefox/accounts-2018.lang)
*PR 6635: Update main.lang (navigation bar update. See notes from Alex (Think not as the strings are in shared file. We could copy and paste them to de, fr files).
*PR 6900: Update main.lang (new strings in footer, language selection)

### CODE REVIEW

**Comment by peiying2** [comment]
@flodolo consider some of these strings are used in several files lately, should we move them to main.lang? These are tagged so I don't know what the behavior would be like if they are not localized first in main.lang

*Get a Firefox Account
*Already have an account?
*Sign In

File: en-US/firefox/new/quantum.lang [hunk 1]
```diff
@@ -69,6 +70,12 @@ Advanced Install Options & Other Platforms
 Advanced install options & other platforms
 
 
+## TAG: fxnew_account_032019
+# Link points to https://support.mozilla.org/kb/refresh-firefox-reset-add-ons-and-settings?
+;;Fix a problem
```
Comment (flodolo) [inline en-US/firefox/new/quantum.lang]: This should be showing up in the errors page?

File: en-US/mozorg/about-2019.lang [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+;<strong>2000</strong> non-employee guests welcomed each year
+<strong>2000</strong> non-employee guests welcomed each year
+
+
+# Link points to https://www.mozilla.org/contact/spaces/berlin/
+;Berlin
+Berlin
+
+
+# The strong tag if you need to put the number in the middle of the phrase
```
Comment (flodolo) [inline en-US/mozorg/about-2019.lang]: What does this comment meant?

File: en-US/main.lang [hunk 3]
```diff
@@ -2,6 +2,25 @@
 Navigation
 
 
+# Footer link points to https://www.mozilla.org/privacy/websites/
+;Website Privacy Notice
+Website Privacy Notice
+
+
+# CTA as in language selection
```
Comment (flodolo) [inline en-US/main.lang]: The 3 from #6900 should be in navigation (it's still a shared file)

File: en-US/mozorg/about-2019.lang [hunk 4]
```diff
@@ -165,7 +161,7 @@ Your voice. Your code. Your support. There are so many ways to join the fight fo
 
 # Link points to https://www.mozilla.org/contribute/
 ;Get Involved
-Get Involved
+Get InvInvolvedInvolvedolved
```
Comment (flodolo) [inline en-US/mozorg/about-2019.lang]: Wut?

Comment (peiying2) [reply en-US/mozorg/about-2019.lang]: This string is in main.lang, I intend to keep it here. Will fix it.

**Comment by peiying2** [comment]
@flodolo this just showed up: https://github.com/mozilla/bedrock/pull/6955/files. Should I include it? Also I am not sure about the Yandex and recommended versions.

File: en-US/mozorg/about-2019.lang (hunk 4 above)
Comment (flodolo) [reply en-US/mozorg/about-2019.lang]: Just to make sure: it's clear that I'm asking why you have "Get InvInvolvedInvolvedolved" as translation?

**Comment by peiying2** [comment]
> I can't spot other issues. I'll merge the langchecker PR, feel free to merge this one when ready (don't forget to rebase, just in case, before adding new locales).

Thanks. I will not leverage strings from other files. It will take too much time, :) and I will risk more conflicts.

**Comment by flodolo** [comment]
> Thanks. I will not leverage strings from other files. It will take too much time, :) and I will risk more conflicts.

I agree, translation memory would help.

But note that the rebase is needed independently, because these are files actively translated.

**Pull request merged**